IPython/core/page.py

```diff
@@ -308,6 +308,13 @@ def get_pager_cmd(pager_cmd=None):
             pager_cmd = os.environ['PAGER']
         except:
             pager_cmd = default_pager_cmd
+    
+    if pager_cmd == 'less':
+        try:
+            os.getenv('LESS').index('-r')
```

Comment:
Catching AttributeError often hides bugs (e.g. if this had a typo `os.geteenv`, it would happily behave as if `-r` is not in LESS`), so it's useful to avoid catching common errors like that which can cause silent hiding of bugs later on.